install_files/ansible-base/roles/upgrade/tasks/0-3pre-upgrade.yml

```diff
@@ -0,0 +1,21 @@
+---
+# This role will upgrade 0.3pre instances to 0.3 instances.
+# It will copy and run the upgrade.py script on the app and monitor servers.
+- name: remove old 0.3 packages to avoid version error
+  apt: name={{ item }} state=absent
+  with_items:
+    - securedrop-app-code
+    - securedrop-grsec
+    - securedrop-ossec-agent
+    - securedrop-ossec-server
```

Comment:
Does this work on both app and mon? For example, `securedrop-ossec-agent` is installed on app but not mon, and `securedrop-ossec-server` is installed on mon but not app. I assume the Ansible apt module does not throw an error because if the package was not installed in the first place, it's `state=absent`, otherwise it will be removed. Is that correct?